unix/mpconfigport_freedos.h

```diff
@@ -36,3 +36,6 @@
 
 // djgpp dirent struct does not have d_ino field
 #undef _DIRENT_HAVE_D_INO
+
+// djgpp errno.h have no ENOTSUP
```

Comment:
Usual idiom to do such things is to write such defined in #ifndef ENOTSUP. (What if next version of djgpp will define it?)